### PR TITLE
Feature/a11y

### PR DESCRIPTION
**Pull request by AnthonyTaylor**

### CODE REVIEW

File: development/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Accessibility
+Accessibility is makimg services that everyone can use regardless of their ability or circumstance.
```
Comment (13enedict) [inline development/accessibility.md]: A slight type "makimg"

File: development/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+**When developing a service keep in mind the following:**
+1. 	All functionality must be usable with the keyboard
+
+ **Example:** pop-up calendars should also let users type in a date
+
+2. HTML is valid (using a [validator service](https://validator.w3.org/))
+    * Have complete start and end tags, are nested, according to their specs, do not contain duplicate attributes, any IDs are unique
+
+3. If the web content sets a timer, the user must have the option to adjust/extend it
```
Comment (13enedict) [inline development/accessibility.md]: I'm not sure I understand, when would you have a timer (that could be adjusted)?

File: development/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+4. A mechanism to allow the users to exit/remove repetitive blocks of content 
+
+ **Example:** Skip to content at the top of the page to bypass common nav bar
+
+5. If an input error is automatically detected, it can be identified and described to the user in text
+
+6. Text can be resized (without assistive technology) up to 200% without loss of content or functionality
```
Comment (13enedict) [inline development/accessibility.md]: I initially read this as, the website should include buttons to resize text, it'd be good to clarify it

File: development/accessibility.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    * Down-event (e.g. mouse button pressing) is not used to execute function OR
+    * Completion is on up-event (e.g. releasing mouse button) and can be aborted/undone OR
+    * Up-event reverses down-event OR
+    * Completing on down-event is essential
+
+8. Keyboard focus is visible. User should be able to identify where they are in a webpage/form without any prior context.
+
+ **Example** When filling in a form, the field in focus should be identifiable, such as with a border
+
+9. Make it easier for users to operate functionality through various inputs beyond keyboard
```
Comment (13enedict) [inline development/accessibility.md]: Is this an actual requirement? It seems very hard to test

File: development/accessibility.md (hunk 2 above)
Comment (AnthonyTaylor) [reply development/accessibility.md]: Fixed on local

Comment (AnthonyTaylor) [reply development/accessibility.md]: In Blue Badge we have added a timer as part of the security updates, this timer needs to be extendable by the form filler (or  at least 20  hours

I'll add this to the page as an example

File: development/accessibility.md (hunk 4 above)
Comment (AnthonyTaylor) [reply development/accessibility.md]: Yes it's a requirement: https://www.w3.org/TR/WCAG21/#input-modalities

File: development/accessibility.md (hunk 3 above)
Comment (AnthonyTaylor) [reply development/accessibility.md]: We have taken this to mean using ctrl / cmd + scroll will keep the content visible without scrolling to the right

I'll explain this too.

**Pull request merged**